Take 2: Med kit rework (fill content, resizing, renaming, etc.)

K-Dynamic opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->

Reopening of #34762.

- Renamed treatment kits to "X med kit" (e.g. brute med kit, airloss med kit, toxin med kit).
- Renamed injectors to "Anti-X injector" (e.g. anti-brute auto-injector, anti-airloss auto-injector, anti-poison auto-injector).
- Increased storage of most med kits (2x4 to 3x4), size unchanged (large)
- Added medical item whitelist to med kits
- New fill contents for every med kit, including new chemical bottles and syringes
- Increased med kit prices to compensate for increased storage and fill
- New cargo order and crate for first aid med kit
- Attribution now directly links to /tg/station med kit resprite commit
- Removed lathe recipe for combat medical kit

### New fills
First aid med kit (3x4 storage, 2x4 large size):
- 2x bruise packs
- 1x ointment
- 1x gauze
- 1x pill canister of seven 10u tricordrazine pills
- 1x 15u epinephrine syringe
- 1x 30u epinephrine bottle

Advanced first aid med kit (3x4 storage, 2x4 large):
- 2x medicated suture
- 2x regenerative mesh
- 1x health analyzer
- 1x 15u dexalin plus and saline syringe
- 1x 30u dexalin plus and saline bottle

Brute med kit (3x4 storage, 2x4 large):
- 1x medicated suture
- 2x bruise packs
- 1x gauze
- 1x blood pack
- 2x pill canisters of five 10u bicaridine pills

Burn med kit (3x4 storage, 2x4 large):
- 1x regenerative mesh
- 3x ointments
- 1x pill canister of seven 10u kelotane pills
- 1x pill canister of five 10u dermaline pills
- 1x 15u sigynate syringe

Airloss med kit (3x4 storage, 2x4 large):
- 1x emergency oxygen tank
- 1x emergency nitrogen tank
- 2x medical masks
- 1x 15u dexalin syringe
- 1x 30u dexalin bottle
- 1x pill canister of seven 10u dexalin pills
- 1x 15u inaprovaline syringe
- 1x 30u inaprovaline bottle
- 1x pill canister of seven 10u inaprovaline pills

Toxin med kit (3x4 storage, 2x4 large):
- 1x ethylredoxrazine syringe
- 1x ethylredoxrazine bottle
- 2x anti-poison auto-injectors
- 1x pill canister of five 10u dylovene pills
- 1x pill canister of five 10u charcoal pills

Radiation med kit (3x4 storage, 2x4 large):
- 1x Geiger counter
- 1x phalanximine syringe
- 1x phalaximine bottle
- 2x anti-rad auto-injectors
- 1x pill canister of seven 10u potassium iodide pills
- 1x pill canister of seven 10u hyronalin pills

Combat med kit (2x4 storage, 2x2 normal):
- 1x medicated suture
- 1x regenerative mesh
- 1x 15u ephedrine syringe
- 1x 15u dexalin plus and saline syringe
- 1x anti-brute auto-injector
- 1x anti-burn auto-injector

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->

Makes med kit contents closer to /tg/station and listinv fills (#19362) by increasing the shape and storage from 2x4 to 3x4, as current med kits are emptied after treating one or two patients. Most maps also never increased the number of med kits in medical storerooms despite the reduction in inventory; it would be easier to modify the fill than map extra kits. Some med kits are 'underpowered' in their current state, with blood bags or tricordrazine pills being left behind.

The advanced first aid med kit no longer has blood packs, instead being replaced with a potent syringe and bottle mix of dexalin plus and saline. The kit also comes with a health analyzer to scan for injuries, making it much better overall than the first aid med kit.

The former oxygen deprivation treatment kit did not provide nitrogen (obvious) and would be looted for the emergency pen. Rather than create a new nitrogen deprivation kit and add extra mapping workload, the new airloss med kit provides both nitrogen and oxygen for all crew members. The emergency pen has been replaced with extra dexalin and inaprovaline to treat multiple crew members suffering from asphyxiation.

The Geiger counter has been added back to the radiation med kit, while ipecac has been removed from the toxin med kit as charcoal is better overall (both in-game and in real life).

The poison auto-injector could be mistaken for actual poison and was renamed to anti-poison auto-injector. Other injectors were renamed to follow the same prefix convention.

The combat med kit is better than other med kits as it stores more space than it occupies (2x4 storage, 2x2 space), causing players to prioritise printing combat med kits over other med kits. The lathe recipe for the combat med kit has been removed to reduce powergaming, while retaining its properties for antagonists and CMOs that begin with it.

## Technical details
<!-- Summary of code changes for easier review. -->
yml and rsi changes

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

### New med kit sizes and fills
![image](https://github.com/user-attachments/assets/4a650557-8cd5-45a6-9ec0-f7930e55c2be)
![image](https://github.com/user-attachments/assets/c494cdb7-2134-4ea3-884a-4ddd526e5bbc)
![image](https://github.com/user-attachments/assets/5ad29efb-9925-4fb1-b9a7-304316b5da5c)
![image](https://github.com/user-attachments/assets/e7bfd486-c761-4ad7-8002-2ad4f21a530f)
![image](https://github.com/user-attachments/assets/c361af0e-8aa0-42d5-92cb-fa0fac81cba3)
![image](https://github.com/user-attachments/assets/ae489d33-d0d9-43fb-88db-f8716563b06e)
![image](https://github.com/user-attachments/assets/66c4990b-f74a-4a92-8c58-de6f3e3ad261)
![image](https://github.com/user-attachments/assets/4acb7cb4-0510-4321-8f87-8baeb5c83434)
![image](https://github.com/user-attachments/assets/4864347f-7bd9-45af-918b-769e8a93180a)

### Medical whitelist restriction
![image](https://github.com/user-attachments/assets/4dd67524-fe9b-45d4-a8f9-9e99809c7d3a)

### New med kit cargo prices
![image](https://github.com/user-attachments/assets/93ee5a42-9ea9-40db-bf64-bed6367d2e03)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->
N/A

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- tweak: Several treatment kits renamed to "X med kit" (e.g. brute med kit, airloss med kit, toxin med kit).
- tweak: Several injectors renamed to "Anti-X injector" (e.g. anti-brute auto-injector, anti-airloss auto-injector, anti-poison auto-injector).
- tweak: Storage of most med kits increased from 2x4 to 3x4, large size unchanged.
- tweak: Med kits now have a medical item whitelist.
- tweak: First aid, advanced first aid, brute, burn, airloss, toxin, radiation, and combat med kits have new fills that increase their healing efficacy.
- tweak: Brute, burn, airloss med kits now cost 1000 spesos. Toxin and radiation med kits now cost 800 spesos. Advanced first aid med kits now cost 2200 spesos.
- add: The first aid kit can now be ordered individually from cargo at 1000 spesos.
- remove: Removed combat med kit recipe from medical techfab.

K-Dynamic committed:
med kit attribution

K-Dynamic committed:
remove combat med kit recipe

K-Dynamic committed:
new med kit fills

K-Dynamic committed:
medical cargo catalog and costs

walksanatora commented:
imma be honest. I dont like the new sprites for the medkit. something about the isometric items is just kinda offputting to me. but I am not art maintainer. also the new fills will definely help early-shift medical and ease pressure off of (possibly non-existant) chemist.

deltanedas commented:
the new sprites are from vg i dont know why they were included instead of doing a resprite of current tg to be cabinet perspective, i think it would look much nicer

K-Dynamic committed:
remove combat med kit error for real

K-Dynamic committed:
Merge branch 'master' into med-kit-fill

K-Dynamic committed:
change bottle IDs

first aid med kit; replace one ointment with gauze

rad med kit; add one phalaximine bottle

sowelipililimute reviewed (changes requested):
While this PR _is_ in line with our chemistry doc on how medicines should be supplied, this isn't really the way we want to be taking FAKs. Rather than supplies of medicine for medical to use, we'd prefer FAKs be oriented for non-medical crew to use, e.g. discrete pouches of "small burn autoinjector, small bit of ointment and use it on the burning person" vs a supply of kelotane and dermaline pills.

You can take this PR in the way of turning the inventory fills you have into crates of medical supplies themed around a certain injury, or you can redo the inventory fills for the above scenario. Either is acceptable.

The creep of 2x4 to 3x4 storage is also a concern.
